web/packages/teleport/src/Navigation/SideNavigation/Search.tsx

```diff
@@ -123,7 +123,7 @@ function SearchContent({
               <SearchResult
                 key={index}
                 subsection={subsection}
-                $active={currentView.route === subsection.route}
+                $active={currentView?.route === subsection.route}
```

Comment:
It might be a good idea adding a comment explaining that this value might be undefined and why to both the function definition and to the place where `currentView` is originally defined. Otherwise it's hard to tell that it can be undefined unless you read through the implementation.

https://github.com/gravitational/teleport/blob/28a68483e0a1edaccbb38281f2f055dbccdb3016/web/packages/teleport/src/Navigation/SideNavigation/Navigation.tsx#L126
https://github.com/gravitational/teleport/blob/28a68483e0a1edaccbb38281f2f055dbccdb3016/web/packages/teleport/src/Navigation/SideNavigation/Navigation.tsx#L158

Reply:
I added a comment for this as part of this PR: https://github.com/gravitational/teleport/pull/47889